### PR TITLE
Speed up parsing of EXPERIMENTOR_OUTPOST.focs.txt

### PR DESCRIPTION
**Pull request by Morlic-fo**
In current master, parsing the experimentor outpost takes around 24 seconds on my system.

This PR removes (hopefully) redundant parentheses to work around the performance issue described in #1824. The effect is that the file parses in only 1.7 seconds. The game starts significantly faster after this bottleneck is removed.

### CODE REVIEW

File: default/scripting/buildings/EXPERIMENTOR_OUTPOST.focs.txt [hunk 1]
```diff
@@ -78,15 +78,15 @@ BuildingType
                 Not OwnedBy affiliation = AnyEmpire
             ]
             activation = Or [
-                Turn high = [[EXPERIMENTOR_SPAWN_START_TURN]]      // always remove lanes before spawn start turn
+                Turn high = ([[EXPERIMENTOR_SPAWN_START_TURN]])      // always remove lanes before spawn start turn
```
Comment (geoffthemedio) [inline default/scripting/buildings/EXPERIMENTOR_OUTPOST.focs.txt:81]: This seems to be doing the opposite of what you say in the summary...

Comment (Morlic-fo) [reply default/scripting/buildings/EXPERIMENTOR_OUTPOST.focs.txt:81]: The same commit removes the parentheses from the macro itself so no nesting level is added here.
I wasn't completely sure if the parentheses are needed in places like this. If they aren't, they should be removed as well.

Comment (geoffthemedio) [reply default/scripting/buildings/EXPERIMENTOR_OUTPOST.focs.txt:81]: I don't see why they would be... the macro should be a complete expression without them. If it parses and runs without error without them, then leave them off.

**Comment by geoffthemedio** [comment]
Seems to work for me...
````
00:41:08.194450 [debug] timer : ScopedTimer.cpp:34 : parse_file "EXPERIMENTOR_OUTPOST.focs.txt" time:    10662 ms
[...]
00:45:02.163627 [debug] timer : ScopedTimer.cpp:34 : parse_file "EXPERIMENTOR_OUTPOST.focs.txt" time:      664 ms
````

File: default/scripting/buildings/EXPERIMENTOR_OUTPOST.focs.txt (hunk 1 above)
Comment (geoffthemedio) [reply default/scripting/buildings/EXPERIMENTOR_OUTPOST.focs.txt:81]: In my tests, it doesn't parse properly without the brackets. Not sure why, but apparently they are needed.

**Pull request merged**